x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LocalPhysicalPlanOptimizerTests.java

```diff
@@ -972,6 +972,57 @@ public void testOutOfRangeFilterPushdown() {
         }
     }
 
+    public void testOutOfRangeFilterForDoubleFieldsPushdown() {
+        var allTypeMappingAnalyzer = makeAnalyzer("mapping-all-types.json", new EnrichResolution());
```

Comment:
Optional: it might make sense to add a variant of `makeAnalyzer` that only takes the mapping. Or make this `ALL_TYPES_MAPPING_ANALYZER` shared (there are other tests building it).